src/wallet/wallet.ts

```diff
@@ -147,6 +160,25 @@ class HathorWalletServiceWallet extends EventEmitter implements IHathorWallet {
     await handleCreate(data.status);
   }
 
+  async onNewTx(newTx: any) {
```

Comment:
Why type `any`? I think it's a defined type that you will receive from the conn object, no?

Reply:
Yea, done in 1e3e15236ba33f9bc5fd3d8ca94cdb32c12ccb56